cognitive/src/main/scala/com/microsoft/azure/synapse/ml/cognitive/MultivariateAnomalyDetection.scala

```diff
@@ -397,6 +348,19 @@ trait MADBase extends HasOutputCol
       |""".stripMargin
   }
 
+  protected def submitDataset(dataset: Dataset[_]): Map[String, JsValue] = {
```

Comment:
Can we rename this method as it does both `submitDataset` and `send request to MVAD`?

Reply:
Fixed